Cargo.toml

```
@@ -17,6 +17,12 @@ lto = "thin"
 [profile.bench]
 lto = "thin"
 
+[profile.release.build-override]
+opt-level = 3
+
+[profile.dev.build-override]
+opt-level = 3
+
```

Why not for all the 4 default profiles? Quoting from [the Cargo book](https://doc.rust-lang.org/cargo/reference/profiles.html) (emphasis mine):

> To compile quickly, **all** profiles, by default, do not optimize build dependencies […]

```suggestion
profile.dev.build-override.opt-level = 3
profile.test.build-override.opt-level = 3
profile.bench.build-override.opt-level = 3
profile.release.build-override.opt-level = 3
```

---

cargo gives me warning when I use that syntax. So I opted for:

```toml
[profile.release.build-override]
opt-level = 3

[profile.dev.build-override]
opt-level = 3

[profile.test.build-override]
opt-level = 3

[profile.bench.build-override]
opt-level = 3
```